source/org/zfin/db/postGmakePostloaddb/1150/ZFIN-8696.csv

```diff
@@ -0,0 +1,1230 @@
+﻿accession,feature_id,fmrel_type,marker_id,symbol,ensdarg
```

Comment:
How did you generate this file? From a SQL query or as part of a load?

Reply:
@cmpich 
This is based on a report that I generated for Ken on ZFIN-8395.  The description of the report is:

"
4a) How many UniProt IDs have Ensembl IDs found in the notes field of "Genomice Feature" records in ZFIN?  4b) How many unique ZFIN gene records are these allele records associated with?

Confused on this one?  Look at the notes field on this page: [ZFIN Feature: sa35159](https://zfin.org/ZDB-ALT-190501-723)     We were unable to assign these alleles to a specific gene in ZFIN but we needed to keep the alleles and assign a gene.  We created a gene record (in this case unm_sa35159), attached the allele and made a standard note as to which Ensembl ID the gene corresponds.  We could attach a UniProt ID to the gene based on this relationship (maybe shakey and would have to discuss).
"

The report itself was attached to that ticket as "8395 - UniProt Data.xlsx" and on the excel sheet labeled "4b"

Reply:
Sounds good.